### PR TITLE
[bitnami/milvus] Fix externalEtcd settings with ssl (bitnami#19983)

### LINKED ISSUE #19983: [bitnami/milvus] cannot startup milvus when connect external etcd server with ssl
### Name and Version

bitnami/milvus:3.1.5

### What architecture are you using?

None

### What steps will reproduce the bug?

1. prepare etcd server certs(ca.pem, etcd-server.pem, etcd-server.key) and client certs(ca.pem, etcd-client.pem, etcd-client.key)
1. startup external etcd server with ssl using ca.pem, etcd-server.pem, etcd-server.key
3. generate milvus-etcd-client-tls-secret using kustomization.yaml
image
4. config milvus values.yaml
image

### Are you using any custom parameters or values?

_No response_

### What is the expected behavior?

milvus related services should startup nornamlly

### What do you see instead?

milvus releated services cannot startup, all failed when try to connect to etcd server

### Additional information

milvus helm chart, we should set enableDefaultInitContainers to true, otherwise cannot run `milvus.prepareMilvusInitContainer`, then the extraConfig cannot merge into milvus.yaml
meanwhile, also will try to run `milvus.waitForETCDInitContainer`, `milvus.waitForKafkaInitContainer`, `milvus.waitForS3InitContainer`.
But if etcd server is configured with tls, the waitForETCDInitContainer try to connect to etcd with the system public certs, not using the etcd client certs, then cannot connect to etcd server, pod will fail

### PR DESCRIPTION
**Pull request by chenraoCR**
### Applicable issues

- fixes #19983

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by carrodher** [comment]
Thank you for initiating this pull request. We appreciate your effort. Just a friendly reminder that it's important to sign your commits. Adding your signature certifies that you either authored the patch or have the necessary rights to contribute the changes. You can find detailed information on how to do this in the [“Sign your work” section of our contributing guidelines](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work).

Feel free to reach out if you have any questions or need assistance with the signing process.

**Comment by chenraoCR** [comment]
@carrodher now it is blocked, what else i need to do

File: bitnami/milvus/Chart.yaml [hunk 1]
```diff
@@ -48,4 +48,4 @@ maintainers:
 name: milvus
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/milvus
-version: 4.2.0
+version: 4.2.1
```
Comment (rafariossaa) [inline bitnami/milvus/Chart.yaml:51]: As you are changing parameters, you should be pushing the major version. In this case it should be `5.0.0`.

**Review by rafariossaa — CHANGES_REQUESTED** [review]
Hi,
  Thanks for adding this. Could you take a look to my comment ?

File: bitnami/milvus/Chart.yaml (hunk 1 above)
Comment (chenraoCR) [reply bitnami/milvus/Chart.yaml:51]: noted

**Review by rafariossaa — DISMISSED** [review]
Thanks.
LGTM

**Review by rafariossaa — APPROVED** [review]

**Pull request merged**